### ISSUE TITLE
black start screen

### ISSUE THREAD

**Issue by David-1684** [endless-sky/endless-sky#10870]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When i start the game, from 0.10.9 release, start screen is empty

### Steps to Reproduce

start the game

### Expected Behavior

i expect game start a usual

### Screenshots

![2024-12-20 00 16 51](https://github.com/user-attachments/assets/da998941-6837-4ecd-bae0-c7cb5f9d4650)
![2024-12-20 00 16 58](https://github.com/user-attachments/assets/7c197076-7c71-428a-b56e-3415913e1eec)

### Operating System

Windows 11. 24H2, 26100.2605

### Game Source

GitHub Releases

### Game Version

0.10.10

**Comment by warp-core** [comment]
Your issue first says you are playing version 0.10.9 but then says you are playing 0.10.10. Can you clarify what version of the game you are using?
Additionally, when did you download it, and how did you extract that archive? Did you place it into an empty directory or merge it with a previous install of the game?

**Comment by David-1684** [comment]
I tried both. 0.10.8 and below no problems. Place in empty directory. When i copy data files to version 0.10.8, its works fine (but without new settings features), so problem is with 'exe' file.

**Comment by tibetiroka** [comment]
Does the same thing happen if you get the game from Steam? Also, what GPU do you have?

**Comment by David-1684** [comment]
GTX 1660 Super + UHD Graphics 730. I dont have Steam account, so i dont check.

**Comment by TheGiraffe3** [comment]
Possibly related to #3885/#5546.

Is this reproducible on 0.10.10/0.10.11 alpha? I don't think we can fix the 0.10.9 executables.

**Comment by David-1684** [comment]
0.10.10 same problem. Continuous build even not load. Check it on another win 11 computer, same thing. Need check win 10.

**Comment by David-1684** [comment]
Check 0.10.10 on Windows 10, build 19045.5247, Intel UHD Graphics 620. Same problem.

**Comment by tibetiroka** [comment]
Do you know if the AutoHDR feature of Windows is enabled? That's been known to crash other games.

**Comment by David-1684** [comment]
Diabled. I check 3 different computers (2  with Win 11, 1 with Win 10). All of them - blank screen.

**Comment by David-1684** [comment]
I try to delete all saves, delete ES folder from AppData, nothing.

**Comment by David-1684** [comment]
[errors.txt](https://github.com/user-attachments/files/18210393/errors.txt)

**Comment by warp-core** [comment]
In about 10 to 15 minutes, can you download the "EndlessSky-win64-continuous.zip" file from the bottom of [this page](https://github.com/warp-core/endless-sky/actions/runs/12431189712?pr=48) and run it like you would the continuous build and tell us what happens, and upload the errors file after running it?

**Comment by David-1684** [comment]
Nothing. Not create errors file, and not create ES folder in AppData.

**Comment by David-1684** [comment]
Not use plugins, no high-dpi.

**Comment by David-1684** [comment]
0.10.10 and 0.10.9 - i manage the problem. There is non english caracters in the path. Continuous build - not start even after i change the folder (Paste it to root directory on C drive).

**Comment by warp-core** [comment]
What are the non-English characters?

**Comment by David-1684** [comment]
In the name of the parent folder

**Comment by TomGoodIdea** [comment]
If the game can't get the path to appdata, it should throw a runtime error. Did you get any message about the path?

**Comment by David-1684** [comment]
No. Maybe you meen this?
[errors.txt](https://github.com/user-attachments/files/18219867/errors.txt)

**Comment by TomGoodIdea** [comment]
No, I don't see anything suspicious there.

What happens if you launch the continuous build? Do you get any error boxes, or just nothing? Could you try launching from cmd?

**Comment by David-1684** [comment]
Nothing...

**Comment by TomGoodIdea** [comment]
Run `sfc /scannow` from a cmd with admin permissions, then reboot.

**Comment by tibetiroka** [comment]
> Run `sfc /scannow` from a cmd with admin permissions, then reboot.

I don't think that's needed; they don't have a broken file system. We did manage to produce some brokenness with unicode characters in certain paths, though we haven't managed to reproduce everything in this issue yet. (And when I say we, I mean @warp-core.)

**Comment by warp-core** [comment]
> In the name of the parent folder. Hebrew.

Could you share which particular characters?

**Comment by David-1684** [comment]
Yes. שונות.

**Comment by tibetiroka** [comment]
Nah, 10861 doesn't fix this. There is a chance that the SDL call there already mishandles the encoding, so that might just make it worse.
